policyengine_us/parameters/gov/states/la/tax/income/exemptions/personal.yaml

```diff
@@ -0,0 +1,26 @@
+description: Louisiana provides this personal exemption amount, based on filing status.
+SINGLE:
+  1983-01-01: 4_500
+SEPARATE:
+  1983-01-01: 4_500
+HEAD_OF_HOUSEHOLD:
+  1983-01-01: 9_000
+JOINT:
+  1983-01-01: 9_000
+WIDOW:
+  1983-01-01: 9_000
+
+metadata:
+  unit: currency-USD
+  period: year
+  breakdown:
+  - filing_status
+  label: Louisiana personal exemption amount
+
+  reference:
+    - title: Louisiana Laws Revised Statutes, Title 47 — Revenue and taxation, RS 47:294 —  Personal exemptions and credit for dependents, A
```

Comment:
law specifies 2,500 amount - need to add that as a comment

Reply:
I didn't see 2500 in the law, could you specify the section?